src/dashboard/pages/login/login_page.py

```diff
@@ -0,0 +1,168 @@
+                                autoFocus=True,
+                            ),
+                            # password input field
+                            dcc.Input(
+                                className=NORMAL_FIELD_CLASS + PASS_FIELD_EXTRA_CLASS,
+                                id="password",
+                                type="password",
+                                placeholder="password",
+                            ),
+                            # login button
+                            html.Button(
+                                className=(
+                                    f'bg-[{colors["dark_purp"]}] w-[40%] p-[10px] ' "rounded-full"
+                                ),
+                                children=[
+                                    html.Div(
+                                        className=f'bg-[{colors["white"]}',
+                                        children=[
+                                            html.P("Login", style={"color": colors["white"]}),
+                                        ],
+                                    )
+                                ],
+                                id="login_button",
+                                n_clicks=0,
+                            ),
+                        ],
+                    ),
+                    # right rectangle
+                    html.Div(
+                        className=f'bg-[{colors["white"]}] flex flex-col items-center'
+                        " justify-center w-[600px] rounded-r-lg",
+                        children=[
+                            html.H1(
+                                className=" mt-4 pd-4 ",
+                                children=[
+                                    "Welcome to GraphIt",
+                                ],
+                            ),
+                            # product logo
+                            html.Img(
+                                src=dash.get_asset_url("logoTransparent.png"),
+                            ),
+                        ],
+                    ),
+                    # error pop-up
+                    html.Div(
+                        id="error_input_message",
+                        className=(
+                            "hidden drop-shadow-2xl fixed bottom-[-100px] "
+                            "left-1/2 transform -translate-x-1/2 h-fit w-fit "
+                            "fit-content bg-red-100 border border-red-400 "
+                            "text-red-700 px-4 py-3 mt-3 rounded"
+                        ),
+                        children=[
+                            html.H2(
+                                className="text-center",
+                                children="Wrong username or password!",
+                            ),
+                        ],
+                    ),
+                ],
+            ),
+        ],
+    )
+
+
+@callback(
+    Output("dcc_location_login", "children"),
```

Comment:
Change this id to something more suitable. Or use the location component in the navbar? If many pages need it, perhaps it should be included in the main layout.